package.json

```diff
@@ -3,43 +3,41 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@types/node": "^16.11.11",
-    "@types/react": "^17.0.37",
+    "@types/node": "^17.0.8",
```

Comment:
Oops, I think this might need to track the current version of Node (16). I think the `index.d.ts` for `@types/node` should specify the Nodejs version.